internal/yggdrasil/yggdrasil.go

```diff
@@ -51,36 +52,38 @@ func NewYggdrasilHandler(deviceRepository *edgedevice.Repository, deploymentRepo
 
 func (h *Handler) GetControlMessageForDevice(ctx context.Context, params yggdrasil.GetControlMessageForDeviceParams) middleware.Responder {
 	deviceID := params.DeviceID
+	logger := log.FromContext(ctx, "DeviceID", deviceID)
 	edgeDevice, err := h.deviceRepository.Read(ctx, deviceID, h.initialNamespace)
 	if err != nil {
 		if errors.IsNotFound(err) {
+			logger.Error(err, "edge device is not found")
```

Comment:
WDYT about changing that to info? It's not a real error and only indication that some object is not in data store.